### ISSUE TITLE
PanelMenu item focus not removed

### ISSUE THREAD

**Issue by geromegrignon** [openng-org/optimus-ui#541]
### Describe the bug

When having multiple panel menus and clicking different items, the focus is not removed. This only happens on Firefox. On Chrome it works.

### Reason for not contributing a PR

- [ ] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://primeng.org/panelmenu#multiple

### Environment

The bug only happens in Firefox. On Chrome it works.

### Angular version

19.2.0

### PrimeNG version

v19

### Steps to reproduce the behavior

1. Go to the PanelMenu docs page on Firefox and open the "Multiple" section (https://primeng.org/panelmenu#multiple)
2. Click on multiple different PanelMenu items in different panel menus
3. See how multiple items stay selected (Left Firefox, Right Chrome)
![Image](https://github.com/user-attachments/assets/00dfb9c4-7857-4a16-ace0-b73b83fab761)

### Expected behavior

The items should not stay focused, like it is in Chrome.

> Migrated from `primefaces/primeng#18007` — originally by `@vdbongard` on 2025-03-31

**Comment by geromegrignon** [comment]
*[Originally posted by `@github-actions[bot]` on 2025-10-15T00:19:42.000Z]*

This issue has been automatically marked as stale due to a lack of activity. **If this issue is still relevant to you, please leave a comment** so we can keep it open. We apologize for not being able to prioritize it sooner. If you have any new information or questions, please share them in your comment!

**Comment by geromegrignon** [comment]
*[Originally posted by `@vdbongard` on 2025-11-07T17:57:12.000Z]*

The bug still exists.
